code/modules/mob/living/living_life.dm

```diff
@@ -2,6 +2,11 @@
 	set waitfor = FALSE
 	set invisibility = 0
 
+	var/signal_result = SEND_SIGNAL(src, COMSIG_LIVING_LIFE, seconds, times_fired)
```

Comment:
This feels like it might add a nonzero amount of overhead, can this be scoped down to a lower type of mob, or does this need to be applied as high as living?

Reply:
simple mobs and humans use it